openedx/core/djangoapps/ace_common/template_context.py

```diff
@@ -64,6 +64,7 @@ def get_base_template_context(site):
         'mobile_store_urls': get_config_value_from_site_or_settings('MOBILE_STORE_URLS', site=site),
 
         # Context processor values for dynamic theming
+        'LMS_ROOT_URL': settings.LMS_ROOT_URL,
```

Comment:
Just for the confirmation...

As we are dealing with multi sites in redhouse
so shouldn't we get it from site configurations?

Reply:
We are overriding the main settings in the site configuration so this will be changed in every site.